hypha/core/templates/core/navigation/primarynav-apply.html

```diff
@@ -1,71 +1,85 @@
 {% load i18n apply_tags heroicons %}
 
-{% if request.user.is_authenticated %}
-    <nav role="navigation" aria-label="Primary" class="w-full">
-        <ul class="nav nav--primary" role="menubar">
-            {% primary_navigation_items request.user as nav_items %}
-            {% for item in nav_items %}
-                <li class="nav__item"
-                    role="presentation"
-                    x-data="{open: false}"
-                    x-on:mouseover.away="open = false"
-                >
-                    {% if item.sub_items %}
-                        <div class="relative inline-block">
-                            <a class="nav__link {% if item.url in request.path %}nav__link--active{% endif %}"
-                               href="{{item.url}}"
-                               aria-label="{% trans "Menu Item" %}"
-                               aria-haspopup="menu"
-                               aria-expanded="false"
-                               role="menuitem"
-                               @mouseover="open = true"
-                               @click.prevent="open = ! open">
-                                {{ item.title }}
-                                <span :class="open ? '[&>svg]:translate-y-1': ''">
-                                    {% heroicon_outline "chevron-down" aria_hidden="true" size=16 stroke_width=3 class="inline align-baseline mr-1 transition-transform" %}
-                                </span>
-                            </a>
-                            <div
-                                x-cloak
-                                x-trap.noautofocus="open"
-                                x-on:keydown.escape="open = false"
-                                x-show="open"
-                                x-transition:enter="transition ease-out duration-250"
-                                x-transition:enter-start="transform opacity-0 scale-95 -translate-y-3"
-                                x-transition:enter-end="transform opacity-100 scale-100 translate-y-0"
-                                x-transition:leave="transition ease-in duration-75"
-                                x-transition:leave-start="transform opacity-100 scale-100 translate-y-0"
-                                x-transition:leave-end="transform opacity-0 scale-95 -translate-y-3"
-                                {% comment %} x-transition {% endcomment %}
-                                @click.outside="open = false"
-                                class="min-w-48 absolute block bg-white shadow-xl z-20 mt-4 border-y rounded-sm"
+<nav
+    aria-label="Primary"
+    x-show="showDesktopMenu || isDesktop"
+    class="fixed md:static top-0 md:top-auto end-0 bottom-0 bg-white z-20 md:z-10 shadow-xl md:shadow-none overflow-y-auto md:overflow-clip"
+>
+    <!-- Mobile Menu Opener -->
+    <button
+        class="absolute top-4 end-4 md:hidden border rounded p-1.5 z-30 hover:bg-slate-100 transition-colors"
```

Comment:
@theskumar Maybe make it `top-6` so the open and close buttons align a bit better?